Homework03

RomanMin opened the pull request:
С коробками пока не закончил, увы. Не успеваю :-(

RomanMin committed:
Homework03

С коробками пока не закончил, увы. Не успеваю :-(

RomanMin committed:
Homework03
 Написал по аналогии с разбором ДЗ

Alexandr056 commented on Java2/src/Lesson03/ArraySwap.java:
```diff
@@ -0,0 +1,24 @@
+
+
+        for (int i = 0; i < array1.length; i++) {
+           if (i == currentPosition) {temp = array1[currentPosition];
```
после { начинайте код с новой строки

RomanMin committed:
Homework04
 Готово только частично, прошу прощения.
 пока подготовил только данные для сортировки. Методы пока изучаю.
